java/S3Sink/cloudformation/msf-deploy.yaml

```diff
@@ -0,0 +1,182 @@
+Description: Simple KDA Flink application
+Parameters:
+  ApplicationName:
+    Type: String
+  FlinkRuntimeEnvironment:
+    Type: String
+    Default: FLINK-1_15
+  CodeBucketArn:
+    Type: String
+  CodeKey:
+    Type: String
+  SecurityGroup:
+    Type: AWS::EC2::SecurityGroup::Id
+  SubnetOne:
+    Type: AWS::EC2::Subnet::Id
+  SubnetTwo:
+    Type: AWS::EC2::Subnet::Id
+  SubnetThree:
+    Type: AWS::EC2::Subnet::Id
+  Parallelism:
+    Type: String
+    Default: 1
+  InputStreamRegion:
+    Type: String
+  InputStreamName:
+    Type: String
+  S3BucketName:
+    Type: String
+    Description: Name of bucket
+  S3Path:
+    Type: String
+    Description: path without leading and  trailing /. example test/out
+
+Resources:
+  FlinkApplication:
+    Type: AWS::KinesisAnalyticsV2::Application
+    Properties:
+      ApplicationName: !Ref ApplicationName
+      RuntimeEnvironment: !Ref FlinkRuntimeEnvironment
+      ServiceExecutionRole: !GetAtt FlinkApplicationServiceExecutionRole.Arn
+      ApplicationConfiguration:
+        ApplicationCodeConfiguration:
+          CodeContent:
+            S3ContentLocation:
+              BucketARN: !Ref CodeBucketArn
+              FileKey: !Ref CodeKey
+          CodeContentType: 'ZIPFILE'
+        VpcConfigurations:
```

Comment:
This example does not require VPC networking.

Reply:
Done.